stargate-starter/src/main/java/io/stargate/starter/Starter.java

```diff
@@ -390,6 +394,7 @@ public void start() throws BundleException {
 
     // Install bundles
     context = framework.getBundleContext();
+    context.addFrameworkListener(new BundleFailureListener());
```

Comment:
should we make it more generic and move this mechanism to the `BaseActivator.java` level?
It will add this fail-fast behavior to all starters (that are using the OSGi `Context`)

Reply:
I did not want to call `System.exit()` from bundles and I did not find any other way to make the process terminate on these errors.